ui/src/components/vault/VaultManagement/AdjustVaultForm.jsx

```diff
@@ -179,19 +181,37 @@ const AdjustVaultForm = ({
 
   const updateLockedDelta = (collAction, collDelta) => {
     if (collAction === 'deposit') {
+      setLockedInputError(null);
       onLockedDeltaChange(AmountMath.add(locked, collDelta));
     }
     if (collAction === 'withdraw') {
-      onLockedDeltaChange(AmountMath.subtract(locked, collDelta));
+      let newAmount;
+      try {
+        newAmount = AmountMath.subtract(locked, collDelta);
```

Comment:
Changing `const newAmount` to `let newAmount` just because JS doesn't have `try` expressions but only statements always bugs me. But oh well.

Reply:
Yea, I could avoid it by stuffing more into the try-block, but I think it's better to isolate the error as specifically as possible